docs/source/getting_started.rst

```diff
@@ -29,4 +29,12 @@ Each time you want to use the package, just activate the conda environment:
 
     $ conda activate gammasim-tools-dev
 
+If you need access to the DB, ask one of the developers, update the login \
+credentials in set_env_db_template.sh and source it to activate the \
+environmental variables:
+
+.. code-block:: console
+
+    $ source set_env_db_template.sh
+
 You are all set now =)
```

Comment:
I suggest to remove the smiley. Sometimes things don't work and I found smiley than always very annoying =)